README.md

```diff
@@ -78,6 +78,37 @@ Once connected, run the following make target:
   make local
 ```
 
+### Running Bulk Indexer
+
+#### Locally
+
+Build the bulk indexer by running the following command
+```
+  make reindex
+```
+Then run the executable  
+```
+  ./reindex
+```
+Please make sure your elasticsearch server is running locally on localhost:9092 and version of the server is 7.10, which is the current supported version.
+
+#### Develop
+
+Navigate to ```cmd/reindex/aws.go``` and update esurl to correct elastic search 7.10 url in develop and then build the bulk indexer by running the following command
```

Comment:
Before starting this process we need to be able to use the dp tool, so basically we should say there are prerequisites to being able to reindex search index on an environment.

This should link off to the relevant document in dp-cli - https://github.com/ONSdigital/dp-cli#build-and-run

Reply:
Done in update 1.